collector/docker/README.md

```diff
@@ -0,0 +1,36 @@
+---
+# Ingest Docker metrics using OTEL Collector's Prometheus receiver
```

Comment:
the title says prometheus receiver, but the first line talks about the dockerstats receiver